helm/ph-ee-engine/values.yaml

```diff
@@ -725,9 +725,9 @@ connector_bulk:
       enable: true
     backpressure:
       enable: false
-    success_threshold_check:
+    completion_threshold_check:
       enable: true
-      success_rate: 95
+      completion_rate: 95
   aws:
     region: "us-east-2"
     access_key: "AKIAX32JM37TZOJ5AKFB"
```

Comment:
Can you remove this access key?

Reply:
Sure let me find default and proper way to do it

Reply:
Created ticket for same https://fynarfin.atlassian.net/browse/PHEE-228